src/astro/files/base.py

```diff
@@ -40,11 +39,11 @@ def __init__(
         )
 
     @property
-    def path(self) -> Any:
+    def path(self) -> str:
```

Comment:
@kaxil does this work for you? Its throwing mypy error for me on my local

Reply:
Works for me